Mathlib/Util/Term/Beta.lean

```diff
@@ -16,7 +16,7 @@ The purpose of this is to support substitutions in notations such as
 `∀ i, beta% p i` so that `p i` gets beta reduced when `p` is a lambda.
 -/
 
-namespace Mathlib.Util.TermBeta
+namespace Mathlib.Util.Term.Beta
```

Comment:
Organizationally, I think it would be better to have all of these in the same file. How about `Mathlib.Util.TermReduce` or similar?